components/builder-web/app/shared/shared.module.ts

```diff
@@ -35,6 +35,7 @@ import { PackageListComponent } from './package-list/package-list.component';
 import { ProjectSettingsComponent } from './project-settings/project-settings.component';
 import { PlatformIconComponent } from './platform-icon/platform-icon.component';
 import { VisibilitySelectorComponent } from './visibility-selector/visibility-selector.component';
+import { StatuspageIndicatorComponent } from './statuspage/statuspage-indicator.component';
```

Comment:
Since this isn't "shared" by multiple modules and multiple instances of it wouldn't make sense, we should just put it into the main `AppModule`.